src/fragments/lib-v1/storage/js/download.mdx

```diff
@@ -125,45 +126,18 @@ import { downloadData } from 'aws-amplify/storage';
 const downloadTask = downloadData({ key, data: file });
 downloadTask.cancel();
 try {
-	await downloadTask.result;
+  await downloadTask.result;
 } catch (error) {
-	if (isCancelError(error)) {
-		// Handle error thrown by task cancellation.
-	}
+  if (isCancelError(error)) {
+    // Handle error thrown by task cancellation.
+  }
 }
 ```
 
 <Callout>
-To get the metadata in result for all read apis we have to configure user defined metadata in CORS
-
-Update your bucket's CORS Policy to look like:
+To get the metadata in result for all APIs you have to configure user defined metadata in CORS.
 
-```json
```

Comment:
Replaced example with just a link to the setup instructions in the getting started guide.